src/core/mcap/python/mcap_bindings.cpp

```
@@ -1,62 +1,13 @@
-    }
-
-    void exit(py::object, py::object, py::object)
-    {
-        recorder_.reset();
-    }
-
-private:
-    std::unique_ptr<core::McapRecorder> recorder_;
-};
-
 PYBIND11_MODULE(_mcap, m)
```

Cppcheck (2.20.0)

[style] The function 'PYBIND11_MODULE' is never used.

(unusedFunction)